src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -1372,9 +1376,11 @@ def __init__(self, config):
     @add_start_docstrings_to_model_forward(ROBERTA_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="Jean-Baptiste/roberta-large-ner-english",
```

Comment:
This looks like a non-organization checkpoint. Do we trust the user to leave the model as is in the future?

Reply:
Our tests will tell I think! Think it's still better then having a checkpoint outputting random outputs